app/create_knights.py

```diff
@@ -0,0 +1,32 @@
+from app.knights import CreateKnights
+
+
+def create_knights(dict_of_knights: dict) -> dict[str, CreateKnights]:
+    knights: dict[int] = dict()
+    knights_value = {}
+    for value_knights in dict_of_knights.values():
+        total_power: list[int] = []
+        total_protection: list[int] = []
+        total_hp: list[int] = []
```

Comment:
Better instead of list use integer and than increase value
```suggestion
        total_power: int = 0
        total_protection: int = 0
        total_hp: int = 0
```